setup.cfg

```diff
@@ -1,3 +1,3 @@
 [tool:pytest]
-addopts = --tb=short --ds=carbondoomsday.settings
-testpaths = carbondoomsday/carbondioxide/tests
+addopts = --tb=short --ds=carbondoomsday.settings --disable-pytest-warnings
+testpaths = ./carbondoomsday/carbondioxide/test/
```

Comment:
Fix them there.. ;)
Related: https://github.com/pytest-dev/pytest/issues/2449 - with that you could filter them with logging filters.